db/migrate/20210119005647_add_created_via_to_incoming_email.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddCreatedViaToIncomingEmail < ActiveRecord::Migration[6.0]
+  def change
+    add_column :incoming_emails, :created_via, :integer, null: true, index: true
```

Comment:
Normally I like indices but I wonder if this is necessary? There don't seem to be any queries on `created_via` and it seems mostly used for debugging?

I don't think we should index it unless we plan on querying it in the future.

Reply:
Hmm yeah I don't plan on querying it for now, will remove.